src/plugins/ui-plugin.client.js

```diff
@@ -9,17 +8,6 @@ import {
   CSSPlugin,
 } from 'gsap';
 
-const hasIntersectionObserverSupport =
-  window &&
-  'IntersectionObserver' in window &&
-  'IntersectionObserverEntry' in window &&
-  'intersectionRatio' in window.IntersectionObserverEntry.prototype;
-
-Vue.use(VueLazyLoad, {
-  lazyComponent: true,
-  observer: hasIntersectionObserverSupport,
-});
-
```

Comment:
We are still using `vue-lazyload`, since it does support SSR, so the component is globally registered here
https://github.com/likecoin/liker-land/blob/cd062e60f72f035b2467fe2f3a76ed8ce2609ba0/src/components/ContentCardWrapper.vue#L2-L20